### ISSUE TITLE
Tasks integration test warns that it will be ignored in the future

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#83777]
Not sure how to fix this one myself as they have a string task type?

```
  workspace-namespace
    Tasks
The task [customTesting, First custom task] uses an undefined task type. The task will be ignored in the future.
Debugger listening on ws://127.0.0.1:5876/59f2f01e-98f7-4174-b3b8-560ff690237f
For help, see: https://nodejs.org/en/docs/inspector
The task [customTesting, First custom task] uses an undefined task type. The task will be ignored in the future.
      ÔêÜ CustomExecution task should start and shutdown successfully (771ms)
```

**Comment by alexr00** [comment]
It's happening because in the test we register a task provider for a type that isn't also registered in a package.json. I'll improve this for November.

**Comment by alexr00** [comment]
Fixed by the work done for #84283!
